tsconfig.json

```diff
@@ -25,12 +25,9 @@
   "exclude": [
     "node_modules",
     "build",
-    "demo",
     "tst",
     "scripts",
     ".storybook",
-    "src/internal",
-    "src/**/*.test.tsx",
```

Comment:
Make a cleanup. There's no `demo`, `src/internal` directory and no `*.test.tsx` file in `src` folder.